prometheus/prometheus.yml

```diff
@@ -0,0 +1,1431 @@
+          ## timestamp order as the collection interval, with the maximum being 1s.
+          ##   ie, when interval = "10s", precision will be "1s"
+          ##       when interval = "250ms", precision will be "1ms"
+          ## Precision will NOT be used for service inputs. It is up to each individual
+          ## service input to set the timestamp at the appropriate precision.
+          ## Valid time units are "ns", "us" (or "µs"), "ms", "s".
+          precision = ""
+
+          ## Log at debug level.
+          # debug = false
+          ## Log only error level messages.
+          # quiet = false
+
+          ## Log target controls the destination for logs and can be one of "file",
+          ## "stderr" or, on Windows, "eventlog".  When set to "file", the output file
+          ## is determined by the "logfile" setting.
+          # logtarget = "file"
+
+          ## Name of the file to be logged to when using the "file" logtarget.  If set to
+          ## the empty string then logs are written to stderr.
+          # logfile = ""
+
+          ## The logfile will be rotated after the time interval specified.  When set
+          ## to 0 no time based rotation is performed.  Logs are rotated only when
+          ## written to, if there is no log activity rotation may be delayed.
+          # logfile_rotation_interval = "0d"
+
+          ## The logfile will be rotated when it becomes larger than the specified
+          ## size.  When set to 0 no size based rotation is performed.
+          # logfile_rotation_max_size = "0MB"
+
+          ## Maximum number of rotated archives to keep, any older logs are deleted.
+          ## If set to -1, no archives are removed.
+          # logfile_rotation_max_archives = 5
+
+          ## Override default hostname, if empty use os.Hostname()
+          hostname = ""
+          ## If set to true, do no set the "host" tag in the telegraf agent.
+          omit_hostname = false
+
+
+        ###############################################################################
+        #                            OUTPUT PLUGINS                                   #
+        ###############################################################################
+
+
+        # Configuration for sending metrics to InfluxDB
+        [[outputs.influxdb_v2]]
+           ## The URLs of the InfluxDB cluster nodes.
+           ##
+           ## Multiple URLs can be specified for a single cluster, only ONE of the
+           ## urls will be written to each interval.
+           ##   ex: urls = ["https://us-west-2-1.aws.cloud2.influxdata.com"]
+           urls = ["$INFLUX_HOST"]
+
+           ## Token for authentication.
+           token = "$INFLUX_TOKEN"
+
+           ## Organization is the name of the organization you wish to write to; must exist.
+           organization = "$INFLUX_ORG"
+
+           ## Destination bucket to write into.
+           bucket = "telegraf"
```

Comment:
The bucket should use an environment variable since you are using a variable for it in your dashboard queries